src/styles/cards.scss

```diff
@@ -0,0 +1,100 @@
+@import "./card-stars";
+
+.card {
+  font-size: 10px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  background: $color-white;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__rating {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-reviews {
+      margin: 0;
+      line-height: 14px;
+    }
+  }
+
+  &__product-image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    line-height: 14px;
+    margin-block: 0 16px;
+    color: $secondary-color-text;
+  }
+
+  &__price {
+    height: min-content;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color-text;
+    }
+
+    &-number {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    padding: 12px 70px;
+    border-radius: 5px;
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    color: $color-white;
+    transition-duration: 300ms;
+
+    &:hover {
+      border: 1px solid $blue-color;
```

Comment:
It's better to not duplicate styles which are not changes on the hover or other animations.